inference-engine/tests/functional/plugin/gpu/shared_tests_instances/single_layer_tests/strided_slice.cpp

```diff
@@ -12,35 +12,50 @@ using namespace LayerTestsDefinitions;
 namespace {
 
 stridedSliceParamsTuple ss_only_test_cases[] = {
+        stridedSliceParamsTuple({ 128, 1 }, { 0, 0, 0 }, { 0, 0, 0 }, { 1, 1, 1 },
+                                { 0, 1, 1 }, { 0, 1, 1 },  { 1, 0, 0 },  { 0, 0, 0 },  { 0, 0, 0 },
+                                InferenceEngine::Precision::FP32, CommonTestUtils::DEVICE_GPU),
+        stridedSliceParamsTuple({ 128, 1 }, { 0, 0, 0 }, { 0, 0, 0 }, { 1, 1, 1},
+                                { 1, 0, 1 }, { 1, 0, 1 },  { 0, 1, 0 },  { 0, 0, 0 },  { 0, 0, 0 },
+                                InferenceEngine::Precision::FP32, CommonTestUtils::DEVICE_GPU),
+        stridedSliceParamsTuple({ 1, 12, 100 }, { 0, -1, 0 }, { 0, 0, 0 }, { 1, 1, 1 },
+                                { 1, 0, 1 }, { 1, 0, 1 },  { 0, 0, 0 },  { 0, 1, 0 },  { 0, 0, 0 },
+                                InferenceEngine::Precision::FP32, CommonTestUtils::DEVICE_GPU),
+        stridedSliceParamsTuple({ 1, 12, 100 }, { 0, 9, 0 }, { 0, 11, 0 }, { 1, 1, 1 },
+                                { 1, 0, 1 }, { 1, 0, 1 },  { 0, 0, 0 },  { 0, 0, 0 },  { 0, 0, 0 },
+                                InferenceEngine::Precision::FP32, CommonTestUtils::DEVICE_GPU),
+        stridedSliceParamsTuple({ 1, 12, 100, 1, 1 }, { 0, -1, 0, 0 }, { 0, 0, 0, 0 }, { 1, 1, 1, 1 },
+                                { 1, 0, 1, 0 }, { 1, 0, 1, 0 },  { },  { 0, 1, 0, 1 },  {},
+                                InferenceEngine::Precision::FP32, CommonTestUtils::DEVICE_GPU),
```

Comment:
NIT: maybe some test where negative number appear in 'end' tensor can also be added?